client/templates/person.js

```diff
@@ -6,9 +6,16 @@ Template.person.events({
       Meteor.call('createPair', this._id, e.target.value);
     }
     return false;
-  },
-  "click .remove-one": function (event) {
-    Meteor.call('unpair', {id: this._id});
-    return false;
+  }
+});
+
+Template.person.helpers({
+  isPaired: function (id) {
+    a = People.findOne(id).pairee;
+    if (a == null) {
+      return false;
+    } else {
+      return true;
+    }
```

Comment:
We can simplify this conditional to `return a != null`. Also, in case you didn't know, the `==` here will cause this to match both `null` and `undefined`.